x-pack/test/functional/apps/dev_tools/searchprofiler_editor.ts

```diff
@@ -148,23 +146,35 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       });
 
       it('returns error if profile is executed with no valid indices', async () => {
-        const input = {
-          query: {
-            match_all: {},
-          },
-        };
-
-        await testSubjects.setValue('indexName', '_all');
-        await aceEditor.setValue(editorTestSubjectSelector, JSON.stringify(input));
+        await PageObjects.searchProfiler.setIndexName('_all');
+        await PageObjects.searchProfiler.setQuery(testQuery);
 
-        await testSubjects.click('profileButton');
+        await PageObjects.searchProfiler.clickProfileButton();
 
         await retry.waitFor('notification renders', async () => {
-          const notification = await testSubjects.find('noShardsNotification');
-          const notificationText = await notification.getVisibleText();
-          return notificationText.includes('Unable to profile');
+          return await PageObjects.searchProfiler.editorHasErrorNotification();
         });
       });
     });
+
+    describe('With a test index', () => {
+      before(async () => {
+        await es.indices.create({ index: testIndex });
+      });
+
+      after(async () => {
+        await es.indices.delete({ index: testIndex });
+      });
```

Comment:
Hi @yuliacech did you give it a run against real serverless project on MKI?
I'm not sure if direct index manipulations will work, if not please add `this.tags(['skiOnMKI'])` to do not run this suite for release quality gates

Reply:
Hey @dmlemeshko, thanks a lot for the review! I'm not familiar with running tests on MKI, would you please point me to some info on how to do that?